src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/base_js.html.twig

```diff
@@ -28,6 +28,7 @@
             var xhr = window.XMLHttpRequest ? new XMLHttpRequest() : new ActiveXObject('Microsoft.XMLHTTP');
             options = options || {};
             options.maxTries = options.maxTries || 0;
+            options.retryDelay = options.retryDelay || 500;
```

Comment:
no need to make this an option IMO. Just change the hardcoded `500` value in the `setTimeout` call.

Reply:
Guess that would be fine since since the maxTries is only used for loading the toolbar data at least as far as i see it. Therefor i went with extending and keeping the default behaviour unless told otherwise. I'll update the pr